content/index.md

```diff
@@ -138,6 +138,8 @@ Spencer Brady
 Joey Lieskovan
 
 Hasan Rahim
+
+Dylan Johnson De Lacy
```

Comment:
Is this their(?) first and last name, or full name? I can't tell.

Reply:
First and Last